beacon_chain/state/helpers.py

```diff
@@ -51,31 +68,34 @@ def get_attestation_indices(crystallized_state,
             )
         )
     )
+
+    attestation_indices = []  # type: List[int]
     if filtered_indices_for_slot:
         attestation_indices = filtered_indices_for_slot[0].committee
 
     return attestation_indices
 
 
-def get_new_recent_block_hashes(old_block_hashes,
-                                parent_slot,
-                                current_slot,
-                                parent_hash):
+def get_new_recent_block_hashes(old_block_hashes: List[Hash32],
+                                parent_slot: int,
+                                current_slot: int,
+                                parent_hash: Hash32) -> List[Hash32]:
     d = current_slot - parent_slot
     return old_block_hashes[d:] + [parent_hash] * min(d, len(old_block_hashes))
 
 
-def get_active_validator_indices(dynasty, validators):
+def get_active_validator_indices(dynasty: int,
+                                 validators: List['ValidatorRecord']) -> List[int]:
     o = []
     for index, validator in enumerate(validators):
         if (validator.start_dynasty <= dynasty and dynasty < validator.end_dynasty):
             o.append(index)
     return o
 
 
-def shuffle(lst,
-            seed,
-            config=DEFAULT_CONFIG):
+def shuffle(lst: List[int],
```

Comment:
The function doesn't seem to care about the contents of the list, so `List[Any]` seems to be more appropriate.